### ISSUE TITLE
link to EET not working

### ISSUE THREAD

**Issue by AjlaaSS** [RoxanneSHS/SandrahEET#101]
last time I communicated with you there, now that link do not work. Hope you rememeber me, I'm Ammanda from your old forum. I did try to email you this year, but I get no response.
Are you still maintain EET or? I see you updated Sandrah. If you are active anywhere it would be nice if we could connect.

Hope to hear from you.
Your Ammanda.
<3

**Comment by RoxanneSHS** [comment]
I have finished my last EET playthrough some months ago and moved on. All I am still doing for modding is some bug fixing if people put stuff into github. Otherwise consider me retired.

**Comment by AjlaaSS** [comment]
Pity it ended way it is.. but all things comes to its end. I can fully understand that you get enough and don't want to use more time , guess it was too much of demanding work and time consuming. We all have life in real to move on, but I can always carry SSD with me and finish it once, whatever my path takes me (as I could be leaving country next year). This time I finally progressed more into BG2, than get really overwhelmed by so lot areas, quests, happenings, NPCs. Took a break in hope I will be able to handle it all later. And there was not you anymore to help like in old times. But I will finish it. Lot time of this year I also was not on PC at all, last few months I was playing BG3, which I found nice but of course I will return to what I really like, which is BG1/2 (EET).

I won't ever forget times when I did come to your forum as really nice times, it was not only having someone willing to help, it was overall nice energy, motivating, all us together, willing to do some more out of game... our talks and forum talks, I will always miss it. Nor i will forget you. Wish you all best.

Let me please know for things I asked, for EET and Install Tool, you do still maintain it or it is run by other people now? As I found it, seems recently updated by looking at dates there, you are also in list of who made it but indeed when I move cursor over your it says "retired". Link here does not work anymore. Is EET still around? I have no idea what was/is happening so if you could let me know.

**Comment by RoxanneSHS** [comment]
I am no longer maintaining the Install Tool since I do not plan to use it myself anymore. I have not monitored EET or any modding activities for some time now, so I have no current information. I enjoyed the whole thing while it lasted but during my last run through the saga I found that there was too little to discover anymore to make it worthwhile. And without a direct motivation one quickly loses the skills and knowledge needed to support others. O, that's that.
